src/spacingData.js

```diff
@@ -1,22 +1,40 @@
+// @flow
 /**
  * Describes spaces between different classes of atoms.
  */
+import type {Measurement} from "./units";
 
-const thinspace = {
+const thinspace: Measurement = {
```

Comment:
I guess there's no reason to restrict these to `unit: "mu"`.  Thanks for switching to `Measurement`.